miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/NotificationDto.java

```diff
@@ -182,4 +184,9 @@ public boolean equals(Object obj) {
   public Optional<String> getLaneContents(int lane) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "lane". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)